src/actions/index.js

```diff
@@ -0,0 +1,24 @@
+import { SEARCH, SET_CURRENT_FILM, SORT_BY } from './constans';
+import fetchFilms from '../Api';
+
+export const getFilms = results => ({
+  type: SEARCH,
+  results,
+});
+
+export const getFilmList = (query, typeOfQuery, dispatch) => {
+  fetchFilms(query, typeOfQuery)
```

Comment:
You could also dispatch some actions for showing loading bar for example: `dispatch(startLoading())`